assets/react-hooks/use-select-with-debounce.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useState, useEffect, useCallback } from '@wordpress/element';
+
+// TODO: Write tests.
```

Comment:
Are you going to add this test?

Reply:
A partial test added here: https://github.com/Automattic/sensei/pull/5104/commits/859125c6185f321bb15a9fd343b20d090c0de92e, but I couldn't make the debounce work yet. So I'll keep this part as TODO for now. 😉